tests/UI/pages/BO/catalog/productsV2/add/index.js

```diff
@@ -124,6 +125,37 @@ class Products extends BOBasePage {
   async goToCatalogPage(page) {
     await this.clickAndWaitForNavigation(page, this.goToCatalogButton);
   }
+
+  /**
+   * Click on new product button
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  async clickOnNewProductButton(page) {
+    await this.waitForSelectorAndClick(page, this.newProductButton);
+
+    return this.elementVisible(page, productsPage.modalCreateProduct, 1000);
+  }
+
+  /**
+   * Choose product type
+   * @param page {Page} Browser tab
+   * @param productType
+   * @returns {Promise<void>}
+   */
+  async chooseProductType(page, productType) {
+    await productsPage.chooseProductType(page, productType);
+    await page.waitForNavigation('networkidle', 2000);
```

Comment:
https://playwright.dev/docs/api/class-page#page-wait-for-navigation

```suggestion
    await page.waitForNavigation({timeout: 2000, waitUntil: 'networkidle'});
```